### DISCUSSION TITLE
Reorganizing repo from single- to multi-project

### DISCUSSION THREAD

**Discussion by LidiaVV** [microsoft/AL-Go#1302] [Q&A] [answered]
Would you please advise if it's possible and what are the steps? Can I preserve folder structure in repo where my PTE apps code's residing?
Thank you.

**Comment by freddydk** [comment, marked as answer]
Single project:
```
.github/*
.AL-Go/*
App1/src/*
App1/app.json
App2/src/*
App2/app.json
```

Multi-project:
```
.github/*
Project1/.AL-Go/*
Project1/AppA/src/*
Project1/AppA/app.json
Project1/AppB/src/*
Project1/AppB/app.json
Project2/.AL-Go/*
Project2/AppX/src/*
Project2/AppX/app.json
Project2/AppY/src/*
Project2/AppY/app.json
```

So...
Move all apps + .AL-Go folder into a project folder.
Create another project folder and copy or create the .AL-Go folder and add other apps.

**Reply by LidiaVV** [reply]
Thank you so much for your reply,
So we do need to move app folders one level down. 
I'll try and comeback if it wont work.
